Merge Conflict/Assets/Scripts/UiManager.cs

```diff
@@ -0,0 +1,186 @@
+/**********************************************************************************************************************
+Name:          UiManager
+Description:   Open and close the screens playfield, settings, level, elements, upgrade.  
+                Hirarchy of the screens:
+                UiManager (wrapper which runs this script; contains open/close button to see them both on every sub-menu)
+                |-Playfield
+                |-Mainmenu (contains all buttons to get to the sub-menues)
+                    |-ButtonOpenSettings
+                    |-ButtonOpenLevel
+                    |-ButtonOpenUpgrade
+                    |-ButtonOpenElements
+                |-Level
+                |-Settings
+                |-Upgrade
+                |-Elements
+                |-ButtonObenMainmenu
+                |-ButtonCloseMainmenu
+                |-ButtonExitGame
+               
+               To get the current state of the menu, call the methode: getMenuVisibility(). It returns true if the menu 
+               is opened; otherwise it returns false.
+
+Author(s):     Markus Haubold
+Date:          2024-02-016
+Version:       V1.0 
+TODO:          - maybe its possible to avoid the switch-case in the screenSwitcher()?!
+               - use Singleton-Pattern?!
```

Comment:
switch case is fine,

implement singleton: https://gamedev.stackexchange.com/a/116010